### DISCUSSION TITLE
添加一个批量下载和附属显示

### DISCUSSION THREAD

**Discussion by blackwhita** [Meloong-Git/PCL#1543] [讨论]
可以进行批量下载，并将选择批量下载的mod放在一个盒子？，从中进行二次选择，同时显示模拟运行冲突选项。
可以在下载mod的旁边加一个附属按钮，可以显示出相应版本的所有已知附属

**Comment by WForst-Breeze** [comment]
整体我没太看懂您想表达什么，我的理解是您想说允许批量下载和安装用户所选择的 mod 吗……？您可以看看 #487 和您的预期是不是差不多一个意思，我的理解您的意思就是方便用户快速下载一批 mod，跟 mod 收藏达到的预期效果应该差不太多

---

> 可以进行批量下载，并将选择批量下载的mod放在一个盒子

没太看懂是什么意思……qwq

> 从中进行二次选择，同时显示模拟运行冲突选项

模拟运行……？感觉没有太大必要qwq……

> 可以在下载mod的旁边加一个附属按钮，可以显示出相应版本的所有已知附属

您看看您的意思是不是跟 #953 差不多
